rfcs/text/i18n.md

```diff
@@ -6,40 +6,38 @@ Introduce first class support for internationalisation and localisation within `
 
 # Basic example
 
-The proposal is to introduce a new component `I18nProvider` that allows our consumers to provide a 
-[`react-i18next`](https://github.com/i18next/react-i18next) compatible<sup>[1] [2]</sup> `t` function.
+The proposal is to create a new `I18nProvider` component that allows carbon components to look 
+up translations by key. We will provide `en-GB` translations but we will accept pull requests to add translations for
+other common languages. We will only add complete locales to the `carbon-react` project. Consumers are able to extend
+existing locales if necessary.
 
 We will need to check each usage of our current `i18n-js` implementation to ensure the:
 * key makes sense;
 * translations are provided in each of our supported languages; and
 * interpolation/plural/formatting works correctly.
 
-`react-i18next` also uses [`Suspense`](https://reactjs.org/docs/concurrent-mode-suspense.html) to allow for async
-translation loading. We don't need to support this within `carbon-react` to adopt this approach, but we will use this in
-the future when we add `loading` states to our components.
-
 We will deprecate our `I18n` component. This component consumes `i18n-js` directly and formats any Markdown content.
 There are better alternatives for Markdown, such as [`react-markdown`](https://github.com/rexxars/react-markdown) which 
 avoids `dangerouslySetInnerHTML`. Our consumers can use `react-markdown` and `react-i18next` in tandem. Removing the
 `I18n` from carbon allows our consumers to patch any Markdown parsing vulnerabilities without requiring a new 
 `carbon-react` release.
 
-We will also deprecate all [`i18n` helpers][3]. Continuing to support these is beyond the scope of a component library
+We will also deprecate all [`i18n` helpers][1]. Continuing to support these is beyond the scope of a component library
 and is better handled by a dedicated library such as [`numeral.js`](http://numeraljs.com/). Removing these will further
 reduce the complexity of this project and allow for more advanced configurations by our consumers. Where we use these
 helpers internally, we will use a dedicated library if appropriate. We will ensure that any new i18n helpers are
 considered as private methods i.e. marked as `@private` or not exported.
 
-Finally, each translation will be overridable by our consumers by providing a string prop to the component. This string 
-can also be provided by `react-i18next`.
+Finally, each translation will be overridable by our consumers by providing a prop to the component or by creating
```

Comment:
I would suggest providing a locale is the main mechanism and providing props is seen more for edge cases where the content is quite different for a particular scenario. Overriding translations via props would work, but wouldn't scale particularly well.